interop-testing/src/main/java/io/grpc/testing/integration/TestServiceClient.java

```diff
@@ -396,9 +396,10 @@ private void runTest(TestCases testCase) throws Exception {
   private class Tester extends AbstractInteropTest {
     @Override
     protected ManagedChannelBuilder<?> createChannelBuilder() {
+      boolean useGeneric = false;
       ChannelCredentials channelCredentials;
       if (customCredentialsType != null) {
-        useOkHttp = false; // Retain old behavior; avoids erroring if incompatible
+        useGeneric = true; // Retain old behavior; avoids erroring if incompatible
```

Comment:
Just checking - is the comment still relevant with this change?

Reply:
Yes. The useOkHttp = false was to enable Netty, to work with these credentials. But it enabled the wrong Netty and so failed at runtime. With this change we now enable the other (shaded) Netty to guarantee the credentials and channel type are compatible.

Reply:
Thanks for the explanation.